internal/refresh.go

```diff
@@ -34,7 +34,6 @@ func RefreshCredentials(oidcClient ssooidciface.SSOOIDCAPI, ssoClient ssoiface.S
 	var roleName *string
 
 	lui, err := readUsageInformation()
-	zap.S().Infof("Attempting to refresh credentials for account [%s] with role [%s]", lui.AccountName, lui.Role)
```

Comment:
Since `readUsageInformation` can return `nil` for `lui`, performing this log statement before doing the `err` check is unsafe